src/open_inwoner/templates/master.html

```diff
@@ -33,7 +33,8 @@
     </head>
 
     <body hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'>
-        {% header categories=menu_categories request=request breadcrumbs=breadcrumbs search_form=search_form has_general_faq_questions=has_general_faq_questions %}
+        {% get_solo 'configurations.SiteConfiguration' as config %}
```

Comment:
This is not necessary, we already load the utils so we have access to the context_processors (like we do above with the colors and below with the `has_general_faq_questions`). So, you can add `show_plans=show_plans` to the header template tag.

Reply:
Heh, @vaszig is correct. Note this get_solo was both my suggestion (because we looked at the other code that did something like that), and I even added the line to the context processor earlier: https://github.com/maykinmedia/open-inwoner/blob/95cbe9dfa17ccc250e0766879ceb0a3f98b84d1d/src/open_inwoner/utils/context_processors.py#L79 :confused:

Reply:
I think I did it correctly now, but keeping this comment for future reference.